### ISSUE TITLE
(CV) cron job sched with whenever  for sending CV requests to queues

### ISSUE THREAD

**Issue by ndushay** [sul-dlss/preservation_catalog#649]
Eventually, set up CV cron job with whenever gem in prod

**Comment by ndushay** [comment]
likely, we won't do this via cron ...

**Comment by ndushay** [comment]
we may do any of the following:
- run all the roots in parallel, effectively starting a job that runs the rake task for a single storage root ... for each of the storage roots.
    - determine the longest time needed for any storage root, and possibly run cron every XXX depending on that interval. 
- switch to using redis to kick off CV for individual druids

**Comment by SaravShah** [comment]
From story time (07/30/18)
Update fixity TTL to be 90 days (instead of 1 week, which is way too short).
Query once per week (?) (cron via whenever) to populate CV queue.
Our hope is that we can fixity check the whole catalog within the fixity TTL (i.e. full fixity check on a quarterly basis).
Julian says other orgs fixity check everything once or twice a year, so quarterly would be totally fine.

**Comment by SaravShah** [comment]
Tested this on stage, Currently Enqueued 9000 objects and is slowly going through all of them!
